pkg/config/env.go

```diff
@@ -209,4 +209,21 @@ var Config = struct {
 	// UI path  => localhost:18000/foo"
 	// API path => localhost:18000/foo/api/v1"
 	WebPrefix string `env:"FLAGR_WEB_PREFIX" envDefault:""`
+
+	// NotifyTimeout - for each notifier that uses http.Client to make requests
+	NotifyTimeout time.Duration `env:"FLAGR_NOTIFY_TIMEOUT" envDefault:"30"`
+	// NotifyNumAttempts - how many attempts should be made for each notification in case of failure
+	NotifyNumAttempts int `env:"FLAGR_NOTIFY_NUM_ATTEMPTS" envDefault:"10"`
+	// NotifyRetryMin - the minimum amount of time that should pass between retry attempts
+	NotifyRetryMin int `env:"FLAGR_NOTIFY_RETRY_MIN" envDefault:"0"`
```

Comment:
if it's `0`, does it mean that no min/max limit? if so, maybe we can add a comment about it.

Reply:
oh, saw the implementation, so it will use the default settings, maybe worthing adding a comment about the default retry settings.

Reply:
I'll clarify that with a comment